rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapStorageProviderImpl.java

```diff
@@ -43,9 +39,13 @@ public void setLockingCap(Coin lockingCap) {
     }
 
     @Override
-    public void save() {
+    public void save(ActivationConfig.ForBlock activations) {
         if (activations.isActive(RSKIP134)) {
-            bridgeStorageAccessor.safeSaveToRepository(LOCKING_CAP.getKey(), getLockingCap().get(), BridgeSerializationUtils::serializeCoin);
+            bridgeStorageAccessor.safeSaveToRepository(
+                LOCKING_CAP.getKey(),
+                getLockingCap(activations).get(),
```

Comment:
This is the kind of thing that shold come up in test, but careful here.

You changed `getLockingCap` to return Optional.empty instead of null. So now what happens when you call `save` with empty instead of null?

Reply:
Done! To avoid an `NoSuchElementException` trying to get the value from `Optional.get()` when is `Optional.empty()`, I used `orElse(null)` method to keep the same behavior.

Reply:
i think its better to keep things as they are on this task and create a new one to approach this kind of things. That way is easier to see the impact of the change

Reply:
@julia-zack, the solution related to `Optional` has already been implemented.

Reply:
Better now, thanks